queries.js

```diff
@@ -75,9 +75,17 @@ async function getCredentialsTypeForRemoteDataObject(subject) {
         ?authenticationConf dgftSec:securityConfiguration/rdf:type ?securityConfigurationType .
     }
   `;
-  await query(q);
   const result = await query(q);
-  return result.results.bindings[0] ? result.results.bindings[0].securityConfigurationType.value : null;
+  const { bindings } = result.results;
```

Comment:
Cool. What you could do too is make the query more precise:
can you try using the expression in the query
```
VALUES ?securityConfigurationType {
    <https://www.w3.org/2019/wot/security#BasicSecurityScheme>
   <https://www.w3.org/2019/wot/security#OAuth2SecurityScheme>
}
```